bip39-lib/src/commonTest/kotlin/cash/z/ecc/android/bip39/MnemonicsTest.kt

```diff
@@ -220,15 +212,14 @@ class MnemonicsTest : BehaviorSpec({
 // Test Utilities
 //
 
-@JsonClass(generateAdapter = true)
-data class TestDataSet(
-    @Json(name = "english") val values: List<List<String>>
-)
-
 fun ByteArray.toHex(): String {
     val sb = StringBuilder(size * 2)
-    for (b in this)
-        sb.append(String.format("%02x", b))
+    for (b in this) {
```

Comment:
Because the Kotlin common implementation is more complex and harder to follow compared to the version relying on the JVM, one very small suggestion would be considering whether we need a test case for the to/from hex methods.  They're doing some non-trivial work, and our test assertions rely on them.

One test case could be a round-trip through toHex and fromHex, asserting the data comes out the same.  Another could be converting bytes to hex and making sure the output matches a known expected value.

That said, if there was a bug here our tests should fail due to how our tests are set up.  So I think we have parity with our JVM tests from before.

Put another way: if you were writing this from scratch I'd say we need a test case. But because it worked before under the JVM, we can be more confident that the conversion here is valid.